libs/cdk/virtual-table/src/services/template-parser/template-parser.service.ts

```diff
@@ -168,10 +169,12 @@ export class TemplateParserService<T> {
         const isCustomKey: boolean = getValidHtmlBooleanAttribute(customKey);
         const canBeAddDraggable: boolean = !(stickyLeft || stickyRight);
         const isModel: boolean = checkValueIsFilled(this.keyMap[key as string]);
+        const forceModel: boolean = getValidHtmlBooleanAttribute(column.forceModel);
```

Comment:
ыыыы, why did you create a separate forceModel field? it could have been just:
```ts
const isModel: boolean = checkValueIsFilled(this.keyMap[key as string]) || forceModel; 
```